lib/redis_snapshot.rb

```diff
@@ -14,9 +14,9 @@ def self.load(redis = Discourse.redis)
     keys = redis.keys
 
     values =
-      redis.pipelined do
+      redis.pipelined do |batch|
```

Comment:
```suggestion
      redis.pipelined do |pipeline|
```

Minor suggestion since [redis-rb](https://github.com/redis/redis-rb)'s docs uses `pipeline`

Reply:
I was thinking about this, I kind of prefer the word batch cause it is a lot more clear in my mind. pipeline feels a bit more open to interpretation. That said not against changing usage, but we should apply it evenly.

Reply:
My vote would be for  `pipeline` since the method being called is already named `pipelined`. Anyway, I'm OK if we go either way.